packages/atst/docs/sdk.md

```diff
@@ -265,24 +263,25 @@ const bigNumberAttestation = stringifyAttestationBytes(
 )
 ```
 
-**Note:** `writeAttestation` already does this for you so this is only needed if using a library other than the attestation station.
+**Note:** `writeAttestation` already does this for you so this is only needed if using a library other than `atst`.
 
 ### React API
 
-For react hooks we recomend using the [wagmi cli](https://wagmi.sh/cli/getting-started) with the [etherscan plugin](https://wagmi.sh/cli/plugins/etherscan) and [react plugin](https://wagmi.sh/cli/plugins/react) to automatically generate react hooks around the attestation station.
+For react hooks we recomend using the [wagmi cli](https://wagmi.sh/cli/getting-started) with the [etherscan plugin](https://wagmi.sh/cli/plugins/etherscan) and [react plugin](https://wagmi.sh/cli/plugins/react) to automatically generate react hooks around AttestationStation.
 
-Use `createKey` and `createValue` to convert your raw keys and values into bytes that can be used in the attestation station contract calls.
-Use `parseString`, `parseBool`, `parseAddress` and `parseNumber` to convert values returned by attestation station to their correct data type.
+Use `createKey` and `createValue` to convert your raw keys and values into bytes that can be used in AttestationStation contract calls.
+Use `parseString`, `parseBool`, `parseAddress` and `parseNumber` to convert values returned by AttestationStation to their correct data type.
 
-For convenience we also [export the hooks](../src/react.ts) here:
+For convenience we also [export the hooks here](../src/react.ts):
 - `useAttestationStationAttestation` - Reads attestations with useContractRead
 - `useAttestationStationVersion` - Reads attestation version
-- `useAttestationStationAttest` - Wraps useContractWrite with attestation station abi calling attest
-- `usePrepareAttestationStationAttest` - Wraps usePrepare with attestation station abi calling attest
+- `useAttestationStationAttest` - Wraps useContractWrite with AttestationStation abi calling attest
+- `usePrepareAttestationStationAttest` - Wraps usePrepare with AttestationStation abi calling attest
 - `useAttestationStationAttestationCreatedEvent` - Wraps useContractEvents for Created events
 
 
 
 ## Tutorial
 
-For a tutorial on using the attestation station in general, see out tutorial as well as other Optimism related tutorials in our [optimism-tutorial](https://github.com/ethereum-optimism/optimism-tutorial/tree/main/ecosystem/attestation-station#key-values) repo.
+For a tutorial on using AttestationStation in general, see our tutorial as well as other Optimism related tutorials in our [optimism-tutorial](https://github.com/ethereum-optimism/optimism-tutorial/tree/main/ecosystem/attestation-station) repo.
```

Comment:
If you wanted you could link to the optimism starter in the tutorial as an example of using attestation station with react

Reply:
Good point. I'll add that. I'll also squash the commits, since I'll need to do that anyway.